lib/ingester.js

```diff
@@ -36,27 +37,46 @@ class Ingester {
         return 'en';
     }
 
+    getIngestionStatus () {
+        const failedProportion = this.assetsCount.failed / this.assetsCount.total;
+        if (failedProportion === 0) {
+            return { 'status': 'OK', 'rc': 0 };
+        }
+        if (failedProportion > config.hatchFailRateThreshold) {
+            return { 'status': 'FAILED', 'rc': 1 };
+        }
+        return { 'status': 'WARNING', 'rc': 13 };
```

Comment:
Out of curiosity, what `rc` means? maybe `exitCode` or `errno` are better names?

Reply:
I'll rename rc to returnCode